specification/compute/resource-manager/readme.md

```diff
@@ -234,31 +233,6 @@ input-file:
 - Microsoft.Compute/preview/2020-10-01-preview/cloudService.json
 ```
 
-### Tag: package-2020-09-30
```

Comment:
We dont have to delete this tag as it will create avocado validation failure and we lost history for generation SDK base on this tag.  so pls keep it or let me know why remove it?

Reply:
I put the tag back, but kept it removed from the SDK generation for the latest version.

Reply:
See our ongoing email thread "Candidate PR to fix breaking changes in compute SDK" for details.